internal/vds/vds_test.go

```diff
@@ -389,6 +389,79 @@ func TestSliceMetadataAxisOrdering(t *testing.T) {
 	}
 }
 
+func TestSliceGeospasial(t *testing.T) {
+	testcases := []struct{
+			name      string
+			direction int
+			lineno    int
+			expected  [][]float64
+	} {
+		{
+			name: "Inline",
+			direction: AxisInline,
+			lineno: 3,
+			expected: [][]float64{{8.0, 4.0}, {6.0, 7.0}},
+		},
+		{
+			name: "I",
+			direction: AxisI,
+			lineno: 1,
+			expected: [][]float64{{8.0, 4.0}, {6.0, 7.0}},
+		},
+		{
+			name: "Crossline",
+			direction: AxisCrossline,
+			lineno: 10,
+			expected: [][]float64{{2.0, 0.0}, {14.0, 8.0}},
+		},
+		{
+			name: "J",
```

Comment:
That method is understandably a bit imprecise. 
```
    auto wp1 = ijkCoordinateTransformer.WorldToVoxelIndex({12, 10, -16});
    std::cout << "Voxel index of world 12 10 -16: "<<wp1[0] << " " << wp1[1] << " " << wp1[2] << "\n";
    auto wp2 = ijkCoordinateTransformer.VoxelIndexToWorld(wp1);
    std::cout << "World of voxel index before: "<<wp2[0] << " " << wp2[1] << " " << wp2[2] << "\n";
```

gives me
```
Voxel index of world 12 10 -16: 3 1 2
World of voxel index before: 12 11 -16
```

Should we mention it nearby that "not all testcases, if you want to implement more, would be this nice"? Like J=1 already breaks the stuff

Reply:
What is imprecise? Why is it understandably so? And what does `j=1` break?   If I change the test to `J = 1` I get `[]float64{0, 3}, []float64{12, 11}`, which is correct